types/finding.ts

```diff
@@ -74,3 +74,17 @@ export interface FindingDeleteRequest {
   risk: string;
   emailAddresses: string[];
 }
+
+export interface AwardFinding {
+  contest: number;
+  handle: string;
+  finding: string;
+  risk: string;
+  score: number;
+  pie: number;
+  split: number;
+  slice: number;
+  award: number;
+  awardCoin: string;
+  awardUSD: number;
```

Comment:
I'm assuming these all preserve backwards compatibility with the CSV/GraphQL version, right? I think that's the best way to leave it for now.

Reply:
Yes, i tried to make sure that for right now it's the same type of data that was used before to make sure it was backwards compatible.